tests-behat/bootstrap/Context.php

```diff
@@ -242,7 +242,7 @@ public function iDontSee($arg1)
      */
     public function labelChangesToNumber()
     {
-        $this->getSession()->wait(5000, '!$("#' . $this->buttonId . '").hasClass("loading")');
+        $this->jqueryWait('!$("#' . $this->buttonId . '").hasClass("loading")');
```

Comment:
`jqueryWait` not needed here stictly, but this way, we wait + some delay for AJAX to stabilize (ie. all requests complete)